exercises/strain/strain_test.go

```diff
@@ -19,6 +19,14 @@ import (
 	"testing"
 )
 
+const targetTestVersion = 1
+
+func TestTestVersion(t *testing.T) {
```

Comment:
My reasoning was that the helper functions belonged to the exercise tests and I wanted to keep all that logic together rather than split it up with the version test. So I figured, get the version test done and out the way before moving onto the main tests.